upass/__main__.py

```diff
@@ -18,6 +18,7 @@
 import urwid
 import pyperclip
 import subprocess
+import xdg
```

Comment:
I would prefer not to introduce another dependency for a fairly minor task. Could you replace this with something stdlib? (Read from `os.environ`, make sure it’s a real path, default to `~/.local/share`)